<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentDialog</class>
 <widget class="QDialog" name="EnvironmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Environement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Scene</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Ambiante</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QDoubleSpinBox" name="ambient_x">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="ambient_y">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="ambient_z">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="fog_enable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Fog</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Couleur</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QDoubleSpinBox" name="fog_x">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="fog_y">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="fog_z">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>DÃ©but</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="fog_start">
        <property name="maximum">
         <double>1024.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Fin</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="fog_end">
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <double>1024.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Frustum</string>
     </property>
     <layout class="QFormLayout" name="formLayout_10">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="znear">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="zfar">
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>512.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>zNear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_39">
        <property name="text">
         <string>zFar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="skybox_enable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Skybox</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QTreeWidget" name="skybox_list">
        <column>
         <property name="text">
          <string>Face</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fichier</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QPushButton" name="skybox_down">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ressource.qrc">
            <normaloff>:/Medias/medias/arrow_down.png</normaloff>:/Medias/medias/arrow_down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skybox_up">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ressource.qrc">
            <normaloff>:/Medias/medias/arrow_up.png</normaloff>:/Medias/medias/arrow_up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="skybox_browse">
          <property name="text">
           <string>Parcourir</string>
          </property>
          <property name="icon">
           <iconset resource="ressource.qrc">
            <normaloff>:/Medias/medias/open.png</normaloff>:/Medias/medias/open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skybox_apply">
          <property name="text">
           <string>Apliquer</string>
          </property>
          <property name="icon">
           <iconset resource="ressource.qrc">
            <normaloff>:/Medias/medias/accept.png</normaloff>:/Medias/medias/accept.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
